app/src/renderer/routes.js

```diff
@@ -80,7 +80,11 @@ export default [
     name: `transactions`,
     component: require(`./components/wallet/PageTransactions`).default
   },
-
+  {
+    path: `/network`,
```

Comment:
This should be the default route

Reply:
interesting idea @fedekunze! i don't want to make this change right now, but we should talk about it at the next team meeting. i've added it to the agenda. 

remember, eventually we will have a dashboard as the default route - but this might be a good interim solution.